service/templates/distros.html

```diff
@@ -52,6 +53,7 @@ <h2 style="display:inline-block; padding-right:15px">Configure
       <input required ng-readonly="!activeDistro.new" id="identifier" name="id" type="text" class="form-control" ng-model="activeDistro._id" placeholder="Unique identifier for this distro">
       <label class="icon fa fa-warning distro-error" ng-show="form.id.$error.required">Distro identifier is required<br></label>
       <label class="icon fa fa-warning distro-error" ng-show="form.id.$dirty && form.id.$error.unsique || (activeDistro.new && form.id.$error.unique)">Distro identifier already exists</label>
+      <label class="icon fa fa-warning" ng-show="containerPoolDistros.indexOf(activeDistro._id) >= 0">Distro is not useable for tasks.</label>
```

Comment:
Could you add a why? Non-admins can see the distro page, and they won't have context to understand this message.